Trading/Exchange/bitmax/bitmax_exchange.py

```diff
@@ -65,6 +66,14 @@ def get_market_status(self, symbol, price_example=None, with_fixer=True):
             self.logger.error(f"Fail to get market status of {symbol}: {e}")
             return {}
 
+    async def get_price_ticker(self, symbol: str, **kwargs: dict):
+        try:
+            ticker = await self.connector.client.fetch_ticker(symbol, params=kwargs)
+            ticker[trading_enums.ExchangeConstantsTickersColumns.TIMESTAMP.value] = time.time()
```

Comment:
No problem with time not in millis ?

Reply:
Timestamp is not used but you're right, we should use millis